ApollosProject/apollos-apps
# fix: Spaces being lost in HTMLView

## DESCRIPTION

### What does this PR do, or why is it needed?

If there is text is blank but has siblings, and those siblings are non-block elements, we should continue to show that text. This will reintroduce spaces that were lost between scripture that looks like this: 

```
<span class=\"wj\">A reed shaken by the wind?</span> <span data-number=\"8\" class=\"v\">
```

### What design trade-offs/decisions were made?

Some snaps changed, but I think they are okay and reflect a more accurate rendering of the HTML. On visual inspection things look normal. 

### How do I test this PR?

I created a story called `spans with spaces in between` to test this issue. 

Also make sure you check out the example story which shows a fairly typical HTMLView. 

### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
